### ISSUE TITLE
Installation on Localhost

### ISSUE THREAD

**Issue by zparnold** [felixarntz/wp-gcs-offload#1]
Hey @felixarntz! Awesome stuff you're working on here. I was thinking of forking the project and seeing if I could made some contributions to what you have going on. Right now I tried installing your plugin on a local installation of WordPress I'm messing around with. Looks like there's a missing file. Is that something you have in another one of your repos, or did I do something wrong?

<img width="1138" alt="screen shot 2016-06-20 at 9 53 14 pm" src="https://cloud.githubusercontent.com/assets/5190984/16215586/7cdcd670-3731-11e6-83be-91c7842525e2.png">

**Comment by felixarntz** [comment]
Hey, thanks for your interest. Please be aware that this plugin is in very early development stages though. However if you wanna contribute, I'd appreciate it!

The problem you're having happens because you probably haven't installed the Composer dependencies yet. Are you familiar with Composer?

**Comment by zparnold** [comment]
Yep! I sure am, and I totally forgot about that! 😄 I just installed it. Looks great!! What else were you thinking of adding at this stage?

**Comment by felixarntz** [comment]
It's mainly about testing. I've been basically developing this blindly up until now. Never connected it to GCS - so yeah, testing (and fixing what's gonna come up) is what needs to be done :)

I've already built a lot of the UI around it all, but never checked the actual functionality. Might sound kind of weird, but that's how I usually work on a new project.

What I already tested is the background processing functionality (which allows you to synchronize large existing media libraries without having to leave the page open for hours).

**Comment by zparnold** [comment]
@felixarntz Sounds exciting! Totally cool on how you build plugins. We all have a process. 👍 
I'm happy to help test the upload to GCloud. I use it a lot for other projects, so I'm familiar with the api and bucket CRUD procedures.

**Comment by felixarntz** [comment]
Very cool! Feel free to work on / test anything that strikes you. :)

**Comment by felixarntz** [comment]
The most critical class in the project is probably the Attachment class. The actual processes with GCS are managed in a separate client class, but the Attachment class handles synchronization of an attachment  between GCS and WordPress.

One goal I have with this plugin is that one should be able to freely choose if an attachment should be only on WordPress, only on GCS or on both. This is available as a general setting (which will be sufficient for the most), but one should still be able to adjust individual attachments as necessary.

And, of course, if an attachment is removed on WordPress, it needs to be removed on GCS as well. I already built most of that functionality.

Do you know whether it's possible to bulk delete multiple attachments from GCS by query? All GCS items get passed the attachment ID as meta, so it would be nice to have a "Clean Bucket" button. There might be edge-cases we cannot foresee where many attachments are deleted in another way than how WordPress usually does it. In that case it would be nice to be able to remove abandoned items from GCS.
